chaco/lod_image_plot.py

```diff
@@ -0,0 +1,322 @@
+from contextlib import contextmanager
+
+from enable.base import intersect_bounds
+from traits.api import (
+    Event, Instance, Int, List, Property,
+    cached_property, on_trait_change
+)
+try:
+    # InterpolationQuality required for Quartz backend only (requires OSX).
+    from kiva.quartz.ABCGI import InterpolationQuality
+except ImportError:
+    pass
+else:
+    KIVA_INTERP_QUALITY = {"nearest": InterpolationQuality.none,
+                           "bilinear": InterpolationQuality.low,
+                           "bicubic": InterpolationQuality.high}
+try:
+    from encore.concurrent.futures.serializer import Serializer
+    from encore.concurrent.futures.enhanced_thread_pool_executor import \
```

Comment:
I think that using Traits Futures is probably preferable: https://github.com/enthought/traits-futures . The `TraitsExecutor` in that library is probably a drop-in replacement (or close to it).